packages/console/src/components/UnsavedChangesAlertModal/index.tsx

```diff
@@ -23,14 +26,23 @@ const UnsavedChangesAlertModal = ({ hasUnsavedChanges }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
 
   useLayoutEffect(() => {
+    function validate(navigator: Navigator): asserts navigator is BlockerNavigator {
```

Comment:
This reminds me that the type from react router don intentionally removed these props while they should exists in browser actually - so `as` is ok here I think? You can double check with @xiaoyijun

Reply:
this is interesting.. so:
- shall we fix the version of rr in `package.json`?
- will it be better to fall back safely, i.e. ignore this feature when no `block` found, instead of failing directly?

Reply:
so `as` is ok?

Reply:
`as` is OK in this case, but out goal is to remove `as`, isn't it?

Reply:
our goal is just to remove unnecessary `as` - not all

Reply:
ok, updated